cmd/notify-mailer/main.go

```diff
@@ -74,6 +78,18 @@ func (m *mailer) ok() error {
 	return nil
 }
 
+func (m *mailer) printStatus(cur, total int, start time.Time) {
+	// Avoid divide by zero, invalid state
+	if total <= 0 || cur < 0 || cur > total {
+		fmt.Printf("\rnotify-mailer: error - invalid cur (%d) or total (%d)\n", cur, total)
+	}
+	completion := (float32(cur) / float32(total)) * 100
+	now := m.clk.Now()
+	elapsed := now.Sub(start)
+	fmt.Printf("\rnotify-mailer: Sending %d of %d [%.2f%%]. Elapsed: %s\n",
```

Comment:
I think with a `\n` at the end of the line, `\r` becomes redundant? In general I'm not a fan of using `\r` for status lines since it makes it hard to look back at history.

What if we turn the Printfs into log.Info's, and default to a stdoutLevel of 6, so that we get both logging to Sumologic and logging to stdout (with timestamps)? Then `-quiet` could just be a matter of adjusting the stdoutLevel.

Also, it would be nice to log the regID and email address(es) for each send, if that's reasonable to do.